index.js

```diff
@@ -0,0 +1,64 @@
+let bookCollection = [];
+let bookCounter = 0;
+const inputForm = document.querySelector('#addBooks');
+const displayList = document.querySelector('#bookList');
+
+const addBooks = (titleValue, authorValue) => {
+  bookCounter += 1;
+  bookCollection = [...bookCollection, { id: bookCounter, title: titleValue, author: authorValue }];
+  localStorage.setItem('awesomeBooksCollection', JSON.stringify(bookCollection));
+};
+
+const showBooksList = (arr) => {
+  if (arr.length <= 0) {
+    document.getElementById('bookList').innerHTML = '<h2> List of Books </h2> <ul><li> EMPTY LIST OF BOOKS <li></ul>';
+  } else {
+    const listBook = arr.map((book) => `
+      <li> '${book.title}' by ${book.author}
+      <button type='button' id='${book.id}' class='remove-button'>Remove</button>
+      </li>`).join('');
+    displayList.innerHTML = `<h2> List of Books </h2> <ul> ${listBook} <ul>`;
+  }
+  displayList.addEventListener('click', (e) => {
+    if (e.target.classList.contains('remove-button')) {
+      // eslint-disable-next-line no-use-before-define
```

Comment:
- [ ] Kindly remove this eslint disable line. It's best to fix the issue in the code.